cluster-autoscaler/cloudprovider/aws/auto_scaling_groups.go

```diff
@@ -352,6 +406,33 @@ func (m *asgCache) DeleteInstances(instances []*AwsInstanceRef) error {
 	return nil
 }
 
+func (m *asgCache) getDescribeAutoScalingGroupResults(commonAsg *asg) (*autoscaling.Group, error) {
```

Comment:
Please remove, we don't need this function.

Reply:
Done. Replaced with existing getAutoscalingGroupByName api